### PR TITLE
created a devskolar benefits section 

### PR DESCRIPTION
**Pull request by shaniassi**
**Changes includes:**
- add: components/Devskolar/Benefits - new page. 
- modify: pages/register - imports a new component.

### CODE REVIEW

**Review by Lemon1903 — CHANGES_REQUESTED** [review]
Alright! The page loads the section without any errors. But the only problem is the section doesn't cover the whole screen's width. 

Here is the screenshot of the problem:
image

Could you also organize the folder structure of the `components/Devskolar` folder as suggested in this [PR](https://github.com/PUP-The-Programmers-Guild/TPGWebsite/pull/31#pullrequestreview-1630087715)

Lastly, there are some utility classes that are not needed to be added since they are the default styling such as `items-start`, `content-start`, `justify-start`, `text-left`, etc.

As for my question, does your tailwindcss class names sort automatically without using a `.prettierrc` file? Since how your utility classes are sorted is the one I'm having before and it changed for I don't know reason. For example your `font-heading` (which is a custom tailwind configuration) comes after `text-center` and before `text-4xl`, but to me it comes first since it is a new added configuration. Weird, since it would cause inconsistent formatting in tailwind utility classes

**Comment by shaniassi** [comment]
Thank you for your feedback; I'll make the necessary changes as soon as I can.  As for your question, I manually organized the classes instead of using Prettier. I'm still new to Tailwind so I didn't think about using Prettier.

**Comment by shaniassi** [comment]
Good evening, I apologize for the late message🥲. I have completed the needed changes to this section, which are fixing the screen's width and removing some utility classes. I just want to know if do I need to create a new pull request to apply these changes?

**Comment by shaniassi** [comment]
[Update]

**Changes includes:**
- fixed the screen's width
- removed some utility classes
- used Prettier to sort classes

**Comment by shaniassi** [comment]
**Changes includes:**
 - Created a folder for benefits section 
`src/
└── components/
    └── Devskolar/ (devskolar page folder)
        ├── Hero/ (hero section folder)
        │   ├── Hero.module.css (components under hero section)
        │   ├── Hero.tsx
        │   └── HeroSteps.tsx
        ├── Process/ (process section folder)
        │   ├── Process.tsx (components under process section)
        │   └── ProcessBox.tsx
        └── Requirements/
            ├── RequirementBars.tsx
            └── Requirement.tsx
        │   └── ProcessBox.tsx
        └── Benefits
            └── Benefits.tsx`

**Review by Lemon1903 — CHANGES_REQUESTED** [review]
This branch now have merge conflicts since there are new changes that has been merged in the `dev` branch. Please resolve those conflicts first.

There are still some unresolved reviews from the last review, please check them out and tell me if you have any problems with that, thanks!

`w-1440` class is not registered in `tailwind.config.js` and it's not an arbitrary value also so it doesn't have any meaning. Just change it to `w-full` for now

The `Benefits.tsx` not having its own folder in Devskolar folder should be fine for now if Benefits section only have 1 file

File: src/components/Devskolar/Benefits.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+export default function DevskolarBenefitsPage() {
+    return (
+      <div className="h-620 w-1440p relative inline-flex items-start justify-start gap-16 px-44 py-10 bg-[#8A8A8A] ">
```
[the remark this answers is not in the document]
Comment (Lemon1903) [reply src/components/Devskolar/Benefits.tsx]: > Also could you replace div tag here to section tag to make it semantically right, thanks!

### Update from the new changes:
`w-1440` class is not registered in `tailwind.config.js` and it's not an arbitrary value also so it doesn't have any meaning. Just change it to `w-full` for now

[the remark this answers is not in the document]
Comment (shaniassi) [reply src/components/Devskolar/Benefits.tsx]: Oh, I see🥲. I'll take note of these, thank you!

**Comment by shaniassi** [comment]
**Changes includes:** 

- Included the w-full and justify-center classes while replacing the div tag with a section tag.
- Replaced the SVG for the right arrow with the Phosphor icon for the right arrow.
-  Placed all four benefits into a single container.

**Review by Lemon1903 — APPROVED** [review]
I added some minor changes to the last commit, mainly just refactoring some unnecessary tw utilities and html tags. Anyways, LGTM!

**Pull request merged**